samples/bluetooth/broadcast_audio_source/src/main.c

```diff
@@ -469,7 +469,7 @@ int main(void)
 	err = usb_enable(NULL);
 	if (err && err != -EALREADY) {
 		printk("Failed to enable USB (%d)", err);
-		return 0;
+		return err;
```

Comment:
As per https://docs.zephyrproject.org/latest/develop/languages/cpp/index.html#enabling-c-support, the `main` function should just return `0`:

> Zephyr ignores the return value from main, so applications should not return status information and should, instead, return zero.

Reply:
Reverted